scripts/microbe_stage/microbe_ai.as

```diff
@@ -403,7 +403,7 @@ class MicrobeAISystem : ScriptSystem{
             MicrobeComponent@ secondMicrobeComponent = cast<MicrobeComponent>(
                 world.GetScriptComponentHolder("MicrobeComponent").Find(allMicrobes[i]));
 
-            if (allMicrobes[i] != microbeEntity && (secondMicrobeComponent.speciesName != microbeComponent.speciesName) && !secondMicrobeComponent.dead){
+            if (allMicrobes[i] != microbeEntity && (secondMicrobeComponent.species.name != microbeComponent.species.name) && !secondMicrobeComponent.dead){
```

Comment:
All of the checks in this file could also probably be made into handle comparisons (that might help quite a bit with performance).